### PR TITLE
calendar-contacts-server: Update to final ccs-calendarserver commit, add TLS updates

### PR DESCRIPTION
**Pull request by essandess**
* Update to final ccs-calendarserver commit of archived repo
* Provide TLS update notes and instructions
* Minor Portfile fixes: variable names and whitespace

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by essandess** [comment]
Note, this is an error of `lint --nitpick`:
```
port lint --nitpick calendar-contacts-server:
Warning: Line 123 has trailing whitespace before newline
```

There is no trailing whitespace.

I do not see the build errors in the CIs that failed.

**Comment by mf2k** [comment]
> Note, this is an error of `lint --nitpick`:
> 
> ```
> port lint --nitpick calendar-contacts-server:
> Warning: Line 123 has trailing whitespace before newline
> ```
> 
> There is no trailing whitespace.
> 
> I do not see the build errors in the CIs that failed.

It's there. It is at the end of the comment line right after the text "if such don't exist".

**Comment by essandess** [comment]
> It's there. It is at the end of the comment line right after the text "if such don't exist".

🤦🏼‍♂️ Thanks.

File: net/calendar-contacts-server/Portfile [hunk 1]
```diff
@@ -1,71 +1,73 @@
...
+                        server implementing the CalDAV and CardDAV \
+                        protocols, including iMIP and APNS. It \
+                        provides a shared location on the network \
+                        allowing multiple users to store and edit \
+                        calendaring and contact information. This port \
+                        provides a basic, working, easily modifiable \
+                        configuration, previously used in macOS \
+                        Server.app, and an nginx reverse proxy to \
+                        handle modern crypto and isolate the backend \
+                        server.
```
Comment (reneeotten) [inline net/calendar-contacts-server/Portfile:31]: please align this the same as the first line on the `long_description` and make white-space changes in a separate commit not mixed in with functional changes

Comment (essandess) [reply net/calendar-contacts-server/Portfile]: Thanks done.

Comment (reneeotten) [reply net/calendar-contacts-server/Portfile]: there are still a lot of other white-space changes and/or reordering of lines happening in the other commit

Comment (essandess) [reply net/calendar-contacts-server/Portfile]: Yes, I see I messed that up.

Comment (essandess) [reply net/calendar-contacts-server/Portfile]: Fixed now.

**Pull request merged**